packages/canon/src/components/Button/styles.css

```diff
@@ -35,18 +35,18 @@
 
   &:hover {
     background-color: transparent;
-    box-shadow: inset 0 0 0 1px var(--canon-outline-focus);
+    box-shadow: inset 0 0 0 1px var(--canon-border-focus);
```

Comment:
Just picking a random spot for this comment really: it seems like there's a good bit of code in here that's not specifically to do with the Canon website. I think it might be worth splitting the change up to make it more digestible, given the overall diff is so large.

Reply:
All the changes made to Canon were actually made for the website to work since we are using these components in practice. I could split it but some part of the site will just not work as expected.